instrumentation-security/cassandra-datastax-4/src/main/java/com/datastax/oss/driver/internal/core/cql/DefaultPrepareRequest_Instrumentation.java

```diff
@@ -0,0 +1,36 @@
+package com.datastax.oss.driver.internal.core.cql;
+
+import com.datastax.oss.driver.api.core.cql.SimpleStatement;
+import com.newrelic.api.agent.security.NewRelicSecurity;
+import com.newrelic.api.agent.security.schema.VulnerabilityCaseType;
+import com.newrelic.api.agent.security.schema.operation.SQLOperation;
+import com.newrelic.api.agent.weaver.MatchType;
+import com.newrelic.api.agent.weaver.Weave;
+import com.newrelic.api.agent.weaver.WeaveAllConstructors;
+import com.newrelic.api.agent.weaver.Weaver;
+import com.newrelic.agent.security.instrumentation.cassandra4.CassandraUtils;
+
+@Weave(type = MatchType.ExactClass, originalName = "com.datastax.oss.driver.internal.core.cql.DefaultPrepareRequest")
+public abstract class DefaultPrepareRequest_Instrumentation {
+    private final SimpleStatement statement = Weaver.callOriginal();
```

Comment:
Please use method weaving only instead of class members

Reply:
Updated in [commit](https://github.com/newrelic/csec-java-agent/pull/72/commits/9b062839c820a747bc98e1df3d6a70b20c15b2b0)